src/sinks/websocket/sink.rs

```diff
@@ -300,6 +302,8 @@ impl WebSocketSink {
                             Ok(())
                         }
                     };
+
+                    drop(finalizers);
```

Comment:
Should this not also update its status on error?

Reply:
Up to you. It looks like this one would be pretty simple except for the config changes.